test/docker-compose.yml

```diff
@@ -8,9 +8,9 @@ services:
       - mysql
     depends_on:
       - mysql
-    restart: unless-stopped
+    restart: always
     volumes:
-      - ./LocalSettings/${LOCALSETTINGS_VARIANT}:/var/www/html/LocalSettings
+      - ./suite-config/${LOCALSETTINGS_VARIANT}:/var/www/html/LocalSettings
```

Comment:
One could argue that this way of solving it has served it's purpose and should be replaced with mounting the desired template over the existing default one. Currently this supports both ways.